groclient/lib.py

```diff
@@ -576,6 +576,39 @@ def get_data_points(access_token, api_host, **selection):
     return list_of_series_to_single_series(resp.json(), False, include_historical)
 
 
+def parse_params(allowed_selections, **selections):
+    params = {}
+    for key, value in list(selections.items()):
+        if key in allowed_selections:
+            params[groclient.utils.str_snake_to_camel(key)] = value
+    return params
+
+
+def get_data_points_v2_prime(access_token, api_host, allowed_selections, **selection):
+    logger = get_default_logger()
+    headers = {'authorization': 'Bearer ' + access_token}
+    url = '/'.join(['https:', '', api_host, 'v2prime/data'])
+    params = parse_params(allowed_selections, **selection)
```

Comment:
checking for required params can be combined with parse_params()
if we need to hardcode the query params in SDK anyway then why keeping 2 copies in both constant.py and experimental.py? we could have simply have one copy in constant and mark the required fields

Reply:
yeah I should do that. I wanted to follow make it similar to `get_data_points` for v2data but it seems thats not the best way.
Good catch, will do.